course/models.py

```diff
@@ -848,33 +860,34 @@ def __unicode__(self):
 
 class FlowRuleException(models.Model):
     flow_id = models.CharField(max_length=200, blank=False, null=False,
-            verbose_name='Flow ID')
+            verbose_name=_('Flow ID'))
     participation = models.ForeignKey(Participation, db_index=True,
-            verbose_name='Participation')
+            verbose_name=_('Participation'))
     expiration = models.DateTimeField(blank=True, null=True,
-            verbose_name='Expiration')
+            verbose_name=_('Expiration'))
 
     creator = models.ForeignKey(User, null=True,
-            verbose_name='Creator')
+            verbose_name=_('Creator'))
     creation_time = models.DateTimeField(default=now, db_index=True,
-            verbose_name='Creation time')
+            verbose_name=_('Creation time'))
 
     comment = models.TextField(blank=True, null=True,
-            verbose_name='Comment')
+            verbose_name=_('Comment'))
 
     kind = models.CharField(max_length=50, blank=False, null=False,
             choices=FLOW_RULE_KIND_CHOICES,
-            verbose_name='Kind')
+            verbose_name=_('Kind'))
     rule = YAMLField(blank=False, null=False,
-            verbose_name='Rule')
+            verbose_name=_('Rule'))
     active = models.BooleanField(default=True,
-            verbose_name=pgettext_lazy(
-                "Is the flow rule exception activated?", "Active"))
+            verbose_name=pgettext_lazy("is the flow rule exception activated?",
+                            "Active"))
```

Comment:
It is context literal, like comments, which won't be show to users. Context are added because there're two "Active"s in the project, one refers to user status, another refers to the state of a exception rule.